### DISCUSSION TITLE
Defining per-metric buckets in Prometheus Exporter/Remote Write sinks

### DISCUSSION THREAD

**Discussion by ivantopo** [vectordotdev/vector#16543] [Q&A] [answered]
Is there a way to configure a custom [buckets](https://vector.dev/docs/reference/configuration/sinks/prometheus_remote_write/#buckets) value for specific metrics in the Prometheus Exporter and/or Remote write sinks?

I'm trying to generate two metrics from log events: one with response times and another with response sizes and these two metrics have pretty different behaviours so fitting both of them on the same bucket configuration doesn't yield good results. I guess I could workaround it by having separate pipelines for each metric with a different remote write sink, but it feels like there has to be a better way. Thanks!

**Comment by StephenWakely** [comment, marked as answer]
>  I guess I could workaround it by having separate pipelines for each metric with a different remote write sink

This is probably the best way at the minute.

We do have an issue on the backlog that addresses this, feel free to add extra context there: https://github.com/vectordotdev/vector/issues/6546

**Reply by ivantopo** [reply]
Thanks for the feedback @StephenWakely!
